src/core/Pricing/PricingCards.tsx

```diff
@@ -45,110 +45,109 @@ const PricingCards = ({
           <Fragment key={title.content}>
             {delimiterColumn(index)}
             <div
-              className={`flex flex-1 p-1 bg-gradient-to-b ${t("from-neutral-1100")} ${t("to-neutral-1100")} rounded-2xl min-w-[272px]`}
+              className={`relative border ${t("border-neutral-1100")} flex-1 px-24 py-32 flex flex-col gap-24 rounded-2xl group ${delimiter ? "@[520px]:flex-row @[920px]:flex-col" : ""} min-w-[272px] overflow-hidden backdrop-blur`}
```

Comment:
Lot of DOM juggling here but the tldr is a rejigging of the backgrounds so that we can support translucency. Alas we lose the gradient border but I made a call to ditch that in favour of the blur as it's more noticable